.g8/inputSelectPage/generated-test/controllers/$package$/$className$ControllerSpec.scala

```diff
@@ -40,7 +40,7 @@ class $className$ControllerSpec extends SpecBase with AppWithDefaultMockFixtures
 
     "must return OK and the correct view for a GET" in {
 
-      when(mock$serviceName$.$lookupReferenceListMethod$(any())).thenReturn(Future.successful($referenceClass;format="decap"$List))
+      when(mock$serviceName$.$lookupReferenceListMethod$(any())).`thenReturn`(Future.successful($referenceClass;format="decap"$List))
```

Comment:
Do we still need the backticks on the thenReturn seeing as we already have the infix approach in use?